### ISSUE TITLE
Unable to handle request using a Proxy Server

### ISSUE THREAD

**Issue by WINOFFRG** [google/shaka-player#3685]
**Have you read the [Tutorials](http://shaka-player-demo.appspot.com/docs/api/tutorial-welcome.html)?**
Yes

**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
Yes

**What version of Shaka Player are you using?**
3.2

**Please ask your question**
I have published an article on [medium](https://winoffrg.medium.com/widevine-license-acquisition-reverse-proxy-server-ac49402d6b4a) which explains how to create a Proxy Server. I need help with this Issue on my [repository](https://github.com/WINOFFRG/widevine-license-proxy-server-nodejs/issues/1).
The issue is also mentioned in the article here 

![image](https://user-images.githubusercontent.com/61311751/136312800-c34e29fb-5b2b-4499-9c33-21d55cc44b09.png)

Basically, if I don't add the `Content-Type` header from the client-side the proxy server fails to parse the raw data in the body.
